db/migrate/20160902111740_initial_schema.rb

```diff
@@ -536,9 +592,9 @@ def create_trigger_to_delete_parent(child_table, parent_table)
       column :text, String, null: false
     end
 
-    # REASONING_STATUSES ||= %w(OPN ERR UNK RSO THM CSA CSAS).freeze
-    # create_enum :reasoning_status_on_conjecture_type,
-    #   [*REASONING_STATUSES, 'CONTR']
+    # PROOF_STATUSES ||= %w(OPN ERR UNK RSO THM CSA CSAS).freeze
```

Comment:
`CONTR` is element of `Conjecture::PROOF_STATUSES`, why the split here?

Reply:
CONTR means that there are proof attempts that say the conjecture is definitely valid and at the same time there are proof attempts that say the conjecture is definitely invalid. So there are contradictory proof attempts. This status does not make sense on a single ProofAttempt (and indicates a bug in the reasoning system).